app/depict/src/main/java/org/openscience/cdk/depict/FreeHepWrapper.java

```diff
@@ -98,13 +116,55 @@ public void dispose() {
         g2.dispose();
     }
 
+    // Documents cited below:
+    // [PLDS92] PostScript Language Document Structuring Conventions Specification, Version 3.0, 25 September 1992
+    // [EGFF96] J.D.Murray & W. vanPyper, Encyclopedia of Graphics File Formats 2nd ed., O'Reilly & Assoc., 1996
```

Comment:
These citations may be better in the class JavaDoc and I would like to suggest to use the @cdk.cite tag for them.